industry_benchmarks/utils/tests/test_plan_rbfe_network.py

```diff
@@ -0,0 +1,64 @@
+import pathlib
+
+import pytest
+from importlib import resources
+import os
+import click.testing
+from ..plan_rbfe_network import run_inputs as main
+from ..plan_rbfe_network import get_settings_charge_changes
+
+@pytest.fixture
+def ligands():
+    with resources.files('utils.tests.data.eg5_inputs') as d:
+        yield str(d / 'ligands_subset.sdf')
+
+
+@pytest.fixture
+def protein():
+    with resources.files('utils.tests.data.eg5_inputs') as d:
+        yield str(d / 'protein.pdb')
+
+
+@pytest.fixture
+def cofactors():
+    with resources.files('utils.tests.data.eg5_inputs') as d:
+        yield str(d / 'cofactor.sdf')
+
+
+@pytest.fixture
+def output():
+    return pathlib.Path('input_jsons')
+
+
+
+class TestScript:
+    def test_invoke(self):
+        runner = click.testing.CliRunner()
+        with runner.isolated_filesystem():
+            result = runner.invoke(main, ["--help"])
+            assert result.exit_code == 0
+            assert "Usage: run-inputs" in result.output
+
+
+    def test_run_inputs(self, ligands, protein, cofactors, output):
+        runner = click.testing.CliRunner()
+        with runner.isolated_filesystem():
+            result = runner.invoke(
+                main,
+                ['--ligands', ligands, '--pdb', protein,
+                 '--cofactors', cofactors, '--output', output]
+            )
+            output_files = list(output.glob("*.json"))
+            assert result.exit_code == 0
+            assert os.path.isdir(output)
+            # Check if the output files are created
+            assert len(output_files) > 0
+
+    def test_get_settings_charge_changes(self):
+        get_settings_charge_changes()
+
+
+    # Test that the number of transformation files for solvent and complex are equal
```

Comment:
Needed tests:
  - Serialize LigandNetwork; check it's there and that the network has the files we need it to have
    - Check the score annotations for at least one edge
    - Check the smcs have partial charges for at least one edge
  - Check the type of files being created (so is there a json and how many)
  - Check the settings in the JSON files (i.e. we should know that there are N json files with charge changes and N without)